src/erlfmt_scan.erl

```diff
@@ -172,21 +178,12 @@ split_shebang(Tokens) ->
         end,
         Tokens
     ),
-    {Shebang, _Anno} = stringify_tokens(ShebangTokens),
+    Shebang = stringify_tokens(ShebangTokens),
     {unicode:characters_to_list(Shebang), Rest}.
 
 %% TODO: make smarter
```

Comment:
Wow can this really be simplified this way.
Is this the something smarter?  Do we still need the comment.

Reply:
I'm not really sure what I had in mind when I wrote that comment 😅
I'll remove it 👍